pkg/rancher-components/src/components/Form/LabeledInput/LabeledInput.vue

```diff
@@ -244,9 +244,12 @@ export default defineComponent({
     /**
      * Emit on input with delay. Note: Arrow function is avoided due context
      * binding.
+     *
+     * NOTE: In multiline, TextAreaAutoGrow emits a string with the value
+     * https://github.com/rancher/dashboard/issues/10249
      */
-    delayInput(event: Event): void {
-      const value = (event?.target as HTMLInputElement)?.value;
+    delayInput(val: string | Event): void {
```

Comment:
I can see that this regressed as a part of some TypeScript refactoring[^1] as a result of to move to `defineComponent`. Where `delyInput` used to always accept a string, but `$event.target.value` isn't valid inside of templates, as `$event` or `target` can be undefined. 

[^1] https://github.com/rancher/dashboard/pull/10226/files#diff-7740b11d8a0c151dff21cdb333af5b5312b35719c169e1938663019db42502e8R248-R250